.github/workflows/k8s_e2e.yml

```diff
@@ -95,15 +95,16 @@ jobs:
           script: |
             // Parameters.
             const minikube_version = [
-              "v1.15.1",
+              "v1.20.0",
             ]
             const kubernetes_version = [
-              { version: "v1.19.2", is_essential: true },
-              { version: "v1.18.9" },
-              { version: "v1.17.12" },
-              { version: "v1.16.15" }, // v1.16.13 is broken, see https://github.com/kubernetes/kubernetes/issues/93194
+              { version: "v1.21.2", is_essential: true },
```

Comment:
Should we mark `is_essential: true` on all supported k8s versions?

Reply:
Maybe? It looks like we were doing it on the "current" version, marking them as essential would make less of a difference between the couple of labels we use to trigger e2e tests though

Reply:
It seems like this just controls the naming of the workflow job? We could drop it then as I don't see why we need it. I could see it if it was controlling which tests were "allowed failures" but currently there don't appear to be any.

Reply:
I suspect when we've determined out "support period" (most recent 3/4, whatever versions) we should have them marked as essential 👍